apps/omg/lib/omg/output/fungible_morevp_token.ex

```diff
@@ -14,26 +14,29 @@
 
 defmodule OMG.Output.FungibleMoreVPToken do
   @moduledoc """
-  Representation of the payment transaction output of a fungible token `currency`
+  Representation of the payment transaction output of a fungible token `currency`.
+  Fungible token outputs could have different `type_marker` but as long as they share the same behaviour
+  they are indistinguishable from code perspective.
   """
   alias OMG.Crypto
-  defstruct [:owner, :currency, :amount]
+  defstruct [:owner, :currency, :amount, :type_marker]
```

Comment:
nitpick: seeing this structure around our channels/docs/code, where the `type_marker` is first  would be nice to help keep the structural consistency in everyone's mind(I understand this isn't necessarily a list either, but for the sake of optics and mental recall). The other option that's also preferrable is let's just start sorting lists like these alphabetically.

Reply:
Why is it called `type_marker`? Isn't it called transaction type / output type everywhere else? What is the meaning of `marker` here? (sorry just a bit confused)

Reply:
While we are on this, we should fix naming for the [other fields too](https://github.com/omisego/plasma-contracts/blob/6ab3525/plasma_framework/docs/integration-docs/integration-doc.md#payment-transaction-format):
* `owner` is `outputGuard`
* `currency` is `token`

Reply:
☝️ both appears in around 100 files - way more this little PR ships. I'd rather create a task to check & align naming convention than try to fix it right away. (Will this be OK @achiurizo ?)

Reply:
Also - regarding tx_type / output_type (markers) I avoid to make them explicit fields unless necessary in flavor of module names.

Reply:
Regarding other renames :point_right: #1210